src/app/interceptors/http-error-handler.ts

```diff
@@ -40,7 +40,7 @@ export class HTTPErrorHandler implements HttpInterceptor {
    */
   displayError(error: HttpErrorResponse) {
     let errorDisplayMsg: string;
-    if (error.error instanceof ErrorEvent) {
+    if (error.error instanceof ErrorEvent || error.status === 0) {
```

Comment:
Should we enhance the error message to suggest that this could be due to a missing root CA?

Reply:
I'd like to keep it short and generic. At least it will allow the user to tell us what the error their getting instead of a blank error message and page.